CARDS-1823: Subject chart layout improvements

marta- opened the pull request:
[Jira link](https://phenotips.atlassian.net/browse/CARDS-1823)

**Preview:**
![image](https://user-images.githubusercontent.com/651980/175036520-76830921-7b28-41d1-a992-ad587a52c91c.png)

marta- committed:
CARDS-1823: Subject chart layout improvements

sashaandjic reviewed (approved):
tested
![chart](https://user-images.githubusercontent.com/23243244/174875073-8af3c297-589f-4c71-846a-8f6e242a7823.png)

marta- committed:
CARDS-1823: Subject chart layout improvements
* Better avatars
* Child subjects are now collapsed by default

marta- commented:
The last commit introduces some avatar improvements and makes child subjects collapsed by default for a cleaner look.

marta- committed:
CARDS-1823: Subject chart layout improvements
Remove large gap between a child subject and its parent

marta- committed:
CARDS-1823: Subject chart layout improvements
Child subjects should not be visible when the parent is collapsed.

marta- committed:
CARDS-1823: Subject chart layout improvements
Child subject header: use a thinner font and move the subject type label outside of the link to be more consistent with how subjects are listed in breadcrumbs

sashaandjic reviewed (approved):
tested - huge improvement!

[before.pdf](https://github.com/data-team-uhn/cards/files/8960251/before.pdf)

[AFTER - all collapsed.pdf](https://github.com/data-team-uhn/cards/files/8960252/AFTER.-.all.collapsed.pdf)
[AFTER - one tumor expanded.pdf](https://github.com/data-team-uhn/cards/files/8960253/AFTER.-.one.tumor.expanded.pdf)
[AFTER - 2 tumors expanded.pdf](https://github.com/data-team-uhn/cards/files/8960256/AFTER.-.2.tumors.expanded.pdf)

veronikaslc commented:
New UI looks much more neat.
